incubator/exporter-otlp/src/main/java/io/aklivity/zilla/runtime/exporter/otlp/internal/duplicated/MetricDescriptor.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.runtime.exporter.otlp.internal.duplicated;
+
+public interface MetricDescriptor
```

Comment:
I think we don't need to use `internalName` terminology here, as the names are the first class names of the metrics we have.

If an exporter needs to externalize the name, thats specific to the exporter, not generic to the engine concept of metrics.

Reply:
I decided to remove this interface altogether (PR #279). It was about to be used by prometheus & otlp exporters, but it turned out they needed different methods after all, so creating this interface wasn't a good abstraction idea.

The actual implementations `OtlpMetricsDescriptor` and `PrometheusMetricDescriptor` indeed use the argument name `internalName`. The reason is that one of the things these classes do is translate the internal metric name to the external metric name, so using just `name` could be ambigious in this context.

Reply:
Right, I was making the point that it is not an internal name, it is just the name, and that anything being exported differently is an externalName or exportedName, so that should be the special case, not internalName.